local/compose/build.go

```diff
@@ -98,7 +98,12 @@ func (s *composeService) ensureImagesExists(ctx context.Context, project *types.
 
 	}
 
-	err := s.build(ctx, project, opts, "auto")
+	mode := "auto"
+	if quietPull {
+		mode = "quiet"
```

Comment:
Little hack to be checked in https://github.com/docker/compose-cli/pull/1390/files#diff-477fda25fb05d92fe2c8e3cb31fdc228a8f5597b9e26b22f7feeff8246404db7R146-R156 to silence the `pull`